src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -230,10 +230,7 @@ export function QueryPageContainerComponent(): JSX.Element {
 
         setIsolateCountStatus(tableResponse.isolateCountStatus);
 
-        if (
-            tableResponse.isolateCountGroups !== undefined &&
-            tableResponse.nrOfSelectedIsolates !== undefined
-        ) {
+        if (tableResponse.nrOfSelectedIsolates !== undefined) {

```

Comment:
it is now the same check as below, you can merge them

Reply:
Done. Commit "Little Changes"